app/routes/content/edit.js

```diff
@@ -71,22 +71,27 @@ export default Route.extend(ConfirmationMixin, Analytics, ResetScrollMixin, Setu
         const isOSF = providerId === 'osf';
 
         // If we're on the proper branded site, stay here.
-        if (themeId === providerId) { return preprint.get('node'); }
+        if (themeId === providerId) { return preprint; }
 
         window.location.replace(`${config.OSF.url}${isOSF ? '' : `preprints/${providerId}/`}${preprint.get('id')}/edit/`);
         return Promise.reject();
     },
 
-    _getContributors(node) {
+    _getContributors(preprint) {
         const controller = this.controllerFor('submit');
-        this.set('node', node);
-        controller.set('node', node);
-        controller.send('getContributors', node);
+        controller.set('model', preprint);
```

Comment:
Only temporarily setting model here to load preprint contributors.  Model is set properly elsewhere.